crates/block-producer/src/test_utils/proven_tx.rs

```diff
@@ -27,6 +27,12 @@ pub struct MockProvenTxBuilder {
     output_notes: Option<Vec<OutputNote>>,
     input_notes: Option<Vec<InputNote>>,
     nullifiers: Option<Vec<Nullifier>>,
+    fee: FungibleAsset,
+}
+
+fn test_fee() -> FungibleAsset {
+    let faucet = ACCOUNT_ID_NATIVE_ASSET_FAUCET.try_into().unwrap();
+    FungibleAsset::new(faucet, 0).unwrap()
 }
```

Comment:
We have this function in multiple places. Maybe there is a way to have just a single copy of it (i.e., in the `test-utils` create)?

Reply:
I moved them into utils for now, to unblock the PR